packages/manager-components/src/components/notifications/ods-notification.tsx

```diff
@@ -26,6 +29,21 @@ const getOdsMessageColor = (type: NotificationType) => {
   }
 };
 
+const getOdsTextColor = (type: NotificationType) => {
```

Comment:
could be a map instead

Reply:
I keep the same structure that the one we have for the message color